backend/database/SQL script/init.sql

```diff
@@ -63,37 +70,37 @@ CREATE TABLE Customers (
     PRIMARY KEY (userId),
 	FOREIGN KEY (userId) REFERENCES Users
 		on DELETE CASCADE
+		on UPDATE CASCADE
 );
 
 CREATE TABLE Riders (
 	userId 		INTEGER,
 	area 		VARCHAR(20) NOT NULL,
-	salary		NUMERIC(8,2),	
-	ratings		INTEGER,
-	type  		VARCHAR(20) NOT NULL,
```

Comment:
where do we keep track of these attributes for riders now?

Reply:
```
WITH RiderDetails AS (
    SELECT R.userId, D.rating, age(D.deliveryTimetoCustomer, D.departTimeForRestaurant) as timeTaken, D.deliveryId
    FROM Riders R left join Delivery_Details D ON R.userId = D.userId
)
SELECT R.userId, coalesce(count(R.deliveryId), 0) as totalOrders, 
                avg(R.timeTaken) as averageTime,
                coalesce(count(R.rating), 0) as totalNoOfRating, 
                coalesce(avg(R.rating), 0) as averageRating
FROM RiderDetails R
GROUP BY R.userId;
```

Reply:
If want for a specific time, we can use `SELECT EXTRACT(MONTH FROM TIMESTAMP '2016-12-31 13:30:15');` and then use a where clause to specify the time period

Reply:
i see, what about the salary attribute? Can't find where its now, or isit meant to be a todo for later?